### PR TITLE
Further update docs to reflect changes kubectl run

### PR DESCRIPTION
**Pull request by janetkuo**
follow up #262 

@kubernetes/docs @pwittrock

### CODE REVIEW

**Comment by janetkuo** [comment]
Published in http://janetkuo.github.io/docs/:

- http://janetkuo.github.io/docs/admin/limitrange/
- http://janetkuo.github.io/docs/admin/namespaces/
- http://janetkuo.github.io/docs/admin/resourcequota/
- http://janetkuo.github.io/docs/user-guide/debugging-services/
- http://janetkuo.github.io/docs/user-guide/docker-cli-to-kubectl/
- http://janetkuo.github.io/docs/user-guide/pods/single-container/

File: docs/admin/limitrange/index.md [hunk 1]
```diff
@@ -95,36 +100,45 @@ were previously created in a namespace.
...
+Let's first spin up a [Deployment](/docs/user-guide/deployments) that creates a single container Pod to demonstrate
 how default values are applied to each pod.
 
 ```shell
 $ kubectl run nginx --image=nginx --replicas=1 --namespace=limit-example
 deployment "nginx" created
+```
+
+Note that `kubectl run` creates a Deployment named "nginx" on Kubernetes cluster >= v1.2. If you are running older versions, it creates replication controllers instead.
+If you want to obtain the old behavior, use `--generator=run/v1` to create replication controllers. See [`kubectl run`](docs/user-guide/kubectl/kubectl_run/) for more details.
```
Comment (bgrant0607) [inline docs/admin/limitrange/index.md]: Does this link need a leading slash?

**Comment by bgrant0607** [comment]
docs/admin/namespaces/ doesn't go to docs/admin/namespaces/index.md?

File: docs/admin/namespaces/index.md [hunk 2]
```diff
@@ -180,6 +148,7 @@ $ kubectl run snowflake --image=kubernetes/serve_hostname --replicas=2
 ```
 We have just created a deployment whose replica size is 2 that is running the pod called snowflake with a basic container that just serves the hostname. 
 Note that `kubectl run` creates deployments only on kubernetes cluster >= v1.2. If you are running older versions, it creates replication controllers instead.
+If you want to obtain the old behavior, use `--generator=run/v1` to create replication controllers. See [`kubectl run`](docs/user-guide/kubectl/kubectl_run/) for more details.
```
Comment (bgrant0607) [inline docs/admin/namespaces/index.md]: Does this link need a leading slash?

**Comment by janetkuo** [comment]
> docs/admin/namespaces/ doesn't go to docs/admin/namespaces/index.md?

No, it's issue #271. Both docs/admin/namespaces/index.md and docs/admin/namespaces.md link to docs/admin/namespaces/ so the latter overwrites the former.

File: docs/admin/limitrange/index.md (hunk 1 above)
Comment (janetkuo) [inline docs/admin/limitrange/index.md]: fixed

File: docs/admin/namespaces/index.md (hunk 2 above)
Comment (janetkuo) [inline docs/admin/namespaces/index.md]: fixed

**Comment by janetkuo** [comment]
>> docs/admin/namespaces/ doesn't go to docs/admin/namespaces/index.md?

>No, it's issue #271. Both docs/admin/namespaces/index.md and docs/admin/namespaces.md link to docs/admin/namespaces/ so the latter overwrites the former. 

This is now solved on HEAD. Rebased and published in http://janetkuo.github.io/docs/:
- http://janetkuo.github.io/docs/admin/limitrange/
- http://janetkuo.github.io/docs/admin/namespaces/walkthrough/
- http://janetkuo.github.io/docs/admin/resourcequota/walkthrough/
- http://janetkuo.github.io/docs/user-guide/debugging-services/
- http://janetkuo.github.io/docs/user-guide/docker-cli-to-kubectl/
- http://janetkuo.github.io/docs/user-guide/pods/single-container/

**Comment by bgrant0607** [comment]
LGTM

**Pull request merged**